runtests.py

```diff
@@ -93,10 +85,12 @@ def run_cmd(name: str) -> int:
     status = 0
     cmd = cmds[name]
     print('run %s: %s' % (name, cmd))
-    res = (system(cmd) & 0x7F00) >> 8
-    if res:
+    proc = subprocess.run(cmd,
+                          stderr=subprocess.STDOUT,
+                          stdout=stdout)
```

Comment:
stdout here is always sys.stdout right? Why do we need to specify this?

Reply:
Removed. I added it while debugging a problem, hoping it would fix it and forgot to remove it after finding the (unrelated) cause of the problem. See https://github.com/python/mypy/pull/10052#issuecomment-782621949 for details.